vizro-core/examples/scratch_dev/app.py

```diff
@@ -1,31 +1,69 @@
 """Dev app to try things out."""
 
-from vizro import Vizro
-import vizro.plotly.express as px
-import vizro.models as vm
-from vizro.tables import dash_ag_grid
 import pandas as pd
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+from typing import List, Literal, Tuple
+from vizro.models.types import ControlType
+from dash import html
 
-df = pd.read_csv("https://raw.githubusercontent.com/plotly/datasets/master/ag-grid/olympic-winners.csv")
-columnDefs = [
-    {"field": "athlete", "headerName": "The full Name of the athlete"},
-    {"field": "age", "headerName": "The number of Years since the athlete was born"},
-    {"field": "country", "headerName": "The Country the athlete was born in"},
-    {"field": "sport", "headerName": "The Sport the athlete participated in"},
-    {"field": "total", "headerName": "The Total number of medals won by the athlete"},
-]
 
-defaultColDef = {
-    "wrapHeaderText": True,
-    "autoHeaderHeight": True,
-}
+class CustomGroup(vm.VizroBaseModel):
+    """Container to group controls."""
+
+    type: Literal["custom_group"] = "custom_group"
+    controls: List[Tuple[str, List[ControlType]]] = [[]]
```

Comment:
FYI my preferred pattern for this would be to nest a list of models, each of which does this (a bit like tabs):

```
    title: str
    controls: list[ControlType] = []
```

Or if you want just one model then do it like this:

```
    controls: dict[str, list[ControlType]] = {}
```

As it stands we made the deliberate choice in vizro to avoid `tuple`. We also haven't used the `dict[str, VizroBaseModel]` pattern anywhere but that might change in future.